std_msgs/String class_name
geometry_msgs/Point32[] bounding_box
geometry_msgs/Point32[] segment
std_msgs/Int32 image_index #Igor
std_msgs/String description #Igor

# Additional fields for spatial tracking
string label
float32 confidence
geometry_msgs/Point position
geometry_msgs/Point size
geometry_msgs/Pose pose
int32[] bbox